.github/workflows/main.yml

```diff
@@ -50,7 +50,6 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - '2.0'
```

Comment:
Shouldn't this have been replaced with Ruby 2.5 since that's what EL8 uses by default?